package.json

```diff
@@ -63,6 +69,7 @@
     "express": "4.15.3",
     "lodash.clonedeep": "4.5.0",
     "lodash.debounce": "4.0.8",
+    "nan": "2.11.1",
```

Comment:
Adding this was the only way I could work around some build issues with errors about 'unresolved node modules'